mail_deduplicate/mail.py

```diff
@@ -199,6 +199,24 @@ def hash_key(self):
         logger.debug(f"Hash: {hash_value}")
         return hash_value
 
+    @cachedproperty
+    def hash_raw_body(self):
+        """ Returns the canonical body hash of a mail. """
+        serialized_raw_body = "\n".join(self.body_lines).encode("utf-8")
```

Comment:
Is there a reason to call `.encode("utf-8")` here? I'd rather have all characters set manipulation/magic happens in the `def body_lines()` above.

Reply:
Well, I tried to be consistent with the `hash_key` property. It uses the `serialized_headers` which is `.encode("utf-8")`.
The `.encode("utf-8")` is only used for `hashlib.sha224` consumption.
Since it was not necessary to extract the `serialized_raw_body` and `serialized_normalized_body` into dedicated properties, it felt more secure to put this `.encode("utf-8")` there than modifying the `body_lines` property.

Reply:
Oh OK, I see. Thanks for the detailed explanation. It's good enough for me. Let's not nitpick on these things! :)